src/makeCalculator.js

```diff
@@ -34,10 +34,36 @@
  * calculator.result === -28
  *
  *
- * @return {object}
+ * @ {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(inputNumber) {
+      return this.result + inputNumber;
+    },
+    subtract(inputNumber) {
+      return this.result - inputNumber;
```

Comment:
```suggestion
    result: 0,
    
    add(inputNumber) {
      return this.result + inputNumber;
    },
    subtract(inputNumber) {
      return this.result - inputNumber;
```
add empty line between methods for better readability